source/Octopus.Tentacle.Client/Scripts/Models/ExecuteKubernetesScriptCommand.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Client.Scripts.Models
+{
+    public class ExecuteKubernetesScriptCommand : ExecuteScriptCommand
+    {
+        public ExecuteKubernetesScriptCommand(
+            ScriptTicket scriptTicket,
+            string taskId,
+            string scriptBody,
+            string[] arguments,
+            ScriptIsolationLevel isolationLevel,
+            TimeSpan isolationMutexTimeout,
+            string isolationMutexName,
+            Dictionary<ScriptType, string>? additionalScripts,
+            ScriptFile[] additionalFiles,
+            string? image,
+            string? feedUrl,
+            string? feedUsername,
+            string? feedPassword)
```

Comment:
nit: I wonder if these could be pulled out into some type that groups them

Reply:
yeah, I can do that 👍

Reply:
done in [8886e99](https://github.com/OctopusDeploy/OctopusTentacle/pull/845/commits/8886e99b2755537ab48270f1a28141562f3ef682)